packages/terra-clinical-action-header/tests/wdio/action-header-spec.js

```diff
@@ -0,0 +1,95 @@
+/* global browser, Terra, before */
+
+const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
+
+describe('Action Header', () => {
```

Comment:
With #280 PR, all the test routes are automatically generated and will need to be updated to follow the generated pattern. This will eliminate the need to hand-write the test routes! (easier for everyone in the long run!)

So these urls need updated:

Reply:
Note: these should not affect the screenshots captured.